microhapulator/cli/mappingqc.py

```diff
@@ -11,40 +11,37 @@
 # -------------------------------------------------------------------------------------------------
 
 import microhapulator.api as mhapi
-import sys
+
+
+def main(args):
+    data = mhapi.read_mapping_qc(
+        args.marker, args.refr, args.rep, figure=args.figure, title=args.title
+    )
+    data.to_csv(args.csv, index=False)
 
 
 def subparser(subparsers):
     desc = "Calculate number of on target, off target, repetitive, and contaminant reads and create a donut plot"
     cli = subparsers.add_parser("mappingqc", description=desc)
     cli.add_argument(
-        "--marker",
+        "marker",
         help="path of csv file containing number of reads mapped to marker sequences",
-        required=True,
     )
     cli.add_argument(
-        "--refr",
+        "refr",
         help="path of csv file containing number of reads mapped to full reference genome",
-        required=True,
     )
     cli.add_argument(
-        "--rep",
+        "rep",
         help="path of csv file containing number of repetitive reads per marker",
-        required=True,
     )
-    cli.add_argument("--csv", help="write read counts to FILE in CSV format", required=True)
+    cli.add_argument("--csv", help="write read counts to FILE in CSV format")
     cli.add_argument(
         "--figure",
         help="create donut plot to FILE showing porportions of on target, off target, repetitive, and contaminant reads",
-        required=True,
     )
     cli.add_argument(
         "--title",
         default=None,
-        help="add a title (such as a sample name) to the histogram plot",
+        help="add a title (such as a sample name) to the donut plot",
```

Comment:
Unless I'm mistaken, this command doesn't make the stacked bar chart, only the donut plots.